modules/packages/conda/metadata.go

```diff
@@ -27,7 +27,7 @@ const (
 	PropertyName     = "conda.name"
 	PropertyChannel  = "conda.channel"
 	PropertySubdir   = "conda.subdir"
-	PropertyMetadata = "conda.metdata"
+	PropertyMetadata = "conda.metadata"
```

Comment:
Is it really a typo? Or it is so, or would it case breaking? @KN4CK3R

Reply:
That's a typo and would be a breaking change. Needs to fixed with a migration.

Reply:
Please advise on how to proceed -- e.g. I could exclude this from being fixed in this PR and you then do in a separate PR if decide to proceed fixing it (note that there were other similar ones like in `committer` IIRC which I had to exclude to not break API)

Reply:
Do you think it is a must to rename it and use a migration to update legacy data?

I haven't read the code carefully, I guess it could be one of these cases:

1. No need to rename it, just leave a comment that "it's a type, but it doesn't affect end users".
2. The typo affects end users, and it must be renamed and get a migration to update legacy data.
3. The typo doesn't affect end user, but it's better to rename it with a migration.

Which case do you think it is?

Reply:
3. The migration is as simple as `UPDATE package_property SET name = 'conda.metadata' WHERE name = 'conda.metdata'`. I will create a PR soonish.